src/ambianic/pipeline/store.py

```diff
@@ -91,22 +91,40 @@ def _save_sample(self,
         inf_json = []
         if inference_result:
             for inf in inference_result:
-                label, confidence, box = inf[0:3]
-                log.info('label: %s , confidence: %.0f, box: %s',
-                        label,
-                        confidence,
-                        box)
-                one_inf = {
-                    'label': label,
-                    'confidence': float(confidence),
-                    'box': {
-                        'xmin': float(box[0]),
-                        'ymin': float(box[1]),
-                        'xmax': float(box[2]),
-                        'ymax': float(box[3]),
+                label, confidence = inf[0:2]
+
+                if isinstance(inf[2], (list, tuple)):
+                    # Process inference for person/object detection model
+                    box = inf[2]
```

Comment:
@ivelin I have written a code to manage inference type whether it contains bounding box or angle difference. As I am not able to see changes in UI, it may fail in UI.

Reply:
@bhavikapanara ok, see my latest comments on how we will solve the UI interface issue. You don't need to spend time figuring out the UI side.